src/index.ts

```diff
@@ -115,16 +115,16 @@ module.exports = class ServerlessPlugin {
     const forwarderConfigs = {
       AddExtension: config.addExtension,
       IntegrationTesting: config.integrationTesting,
-      SubToApiGatewayLogGroup: config.subscribeToApiGatewayLogs,
-      SubToHttpApiLogGroup: config.subscribeToHttpApiLogs,
-      SubToWebsocketLogGroup: config.subscribeToWebsocketLogs,
+      SubToAccessLogGroups: config.subscribeToAccessLogs,
+      SubToExecutionLogGroups: config.subscribeToExecutionLogs,
     };
 
     const defaultRuntime = this.serverless.service.provider.runtime;
     const handlers = findHandlers(this.serverless.service, config.exclude, defaultRuntime);
 
     let datadogForwarderArn;
     datadogForwarderArn = setDatadogForwarder(config);
+    this.serverless.cli.log("Setting Datadog Forwarder");
```

Comment:
What is this log line for? It looks like it will be printed in all cases, even if `datadogForwarderArn` isn't actually set to anything.

Reply:
I don't recall adding this. Will remove.